### DISCUSSION TITLE
How snapshot revision number calculated?

### DISCUSSION THREAD

**Discussion by zxcat** [doublecmd/doublecmd#1283] [General]
There are builds with **revision number** on [snapshots page](https://doublecmd.sourceforge.io/site/rus/snapshots.php). But bugs being fixed in commits, which identified by hash instead of rev.number.
How can I detect is snapshot includes fix I want to test? E.g. I got email referencing commit bba5731 and try to understand is revision `10927` includes it. Seems not, based on time.

Or it would be helpful to add commit id to snapshot name.

**Comment by alexx2000** [comment]
It is a number of commits. You can check [changelog.txt](https://doublecmd.sourceforge.io/snapshots/changelog.txt) to view changes.

**Reply by zxcat** [reply]
thank you!
missed `changelog.txt`, it contains all I need
